src/libraries/System.Private.CoreLib/src/System/Text/Encoding.cs

```diff
@@ -1015,8 +1038,18 @@ public virtual string GetString(byte[] bytes)
         // Returns an encoding for the UTF-7 format. The returned encoding will be
         // an instance of the UTF7Encoding class.
 
+        [Obsolete(Obsoletions.SYSTEM_TEXT_ENCODING_UTF7_MESSAGE, DiagnosticId = Obsoletions.SYSTEM_TEXT_ENCODING_UTF7_DIAGID, UrlFormat = Obsoletions.SHARED_URL_FORMAT)]
         public static Encoding UTF7 => UTF7Encoding.s_default;
 
+        internal static bool IsUTF7EncodingEnabled
```

Comment:
>IsUTF7EncodingEnabled [](start = 29, length = 21)

please move this to [https://github.com/dotnet/runtime/blob/master/src/libraries/System.Private.CoreLib/src/System/LocalAppContextSwitches.cs](https://github.com/dotnet/runtime/blob/master/src/libraries/System.Private.CoreLib/src/System/LocalAppContextSwitches.cs)